py-polars/polars/internals/series/series.py

```diff
@@ -957,6 +957,26 @@ def _repr_html_(self) -> str:
         """Format output data in HTML for display in Jupyter Notebooks."""
         return self.to_frame()._repr_html_(from_series=True)
 
+    def item(self) -> Any:
+        """
+        Return the series as a scalar.
+
+        Equivalent to ``df[0]``, with a check that the shape is (1,).
```

Comment:
A Seires shouldn't be called a DataFrame: --> `s[0]`

Reply:
Changed.